install/local/dev/postgres/Dockerfile

```diff
@@ -7,22 +7,22 @@ ARG ENVIRONMENT="local"
 ARG FILES_DOMAIN="wjfiles.test"
 ARG PGDATA="/var/lib/postgresql/data"
 
-ENV POSTGRES_CLUSTER="main"
-ENV POSTGRES_VERSION="13"
+# Install dependencies
+RUN apk add --no-cache musl-locales
 
 # Copy sources
 COPY ./install/files/postgres /src
 WORKDIR /src
 
 # Create system user
-RUN useradd wikijump --create-home --shell /bin/bash
+RUN adduser -S wikijump
 
 # Setup database
 USER postgres
-RUN pg_createcluster --user postgres --locale en_US.UTF-8 "${POSTGRES_VERSION}" "${POSTGRES_CLUSTER}"
+RUN initdb -D ${PGDATA} --locale=en_US.UTF-8
```

Comment:
Out of curiosity, why does a database have a locale? lol

Reply:
Mostly for string encoding